client/src/app/signup/Form.tsx

```diff
@@ -117,8 +118,10 @@ export default function Form() {
           type='password'
           id='password'
           name='password'
+          pattern='^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*]).{8,}$'
```

Comment:
Creo que acá no estamos contemplando el punto (`.`) o el guión (`-`) como caracteres especiales, no? Puede que se les haya pasado porque lo agregamos después, en #33.

Esta es la regex del BE:
```ruby
/(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-.])/
```

Reply:
Creo que falta la cantidad de caracteres en esa no? creo que serian 8 minimo puede ser?

Reply:
Si si, en esa que pasé falta el control de caracteres, porque eso se controla desde otro lado en el BE.

Básicamente, solo faltarían agregar el punto y el guión a la regex que pusiste 🙂